package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maskbook",
-  "version": "1.10.7",
+  "version": "1.10.8",
```

Comment:
Why bump version here?

Reply:
this branch is forked from tag 1.10.8 because the master branch is broken